kotlin/src/test/KotlinExample.kt

```diff
@@ -100,11 +99,12 @@ class KotlinExample {
     }
 }
 
-fun main(args: List<String>) {
-    val x = KotlinExample()
-    val dash = x.findDashboardsByTitle("Test")
-    val tile = x.getDashboardTile(dash[0], "Test")
-    if (tile != null) {
-        val down = x.downloadTile(tile, "txt")
-    }
-}
+// TODO convert this to a functional test
```

Comment:
TODO

Reply:
Yup. `TODO` is valid for this stage of the KotlinSDK